changelogs/unreleased/gh-6085-log-and-limit-iteration.md

```diff
@@ -0,0 +1,3 @@
+## feature/box
+
+* Introduced fiber slice which allows to limit iteration in space (gh-6085).
```

Comment:
Strictly speaking, this is a breaking change. Please mention it in the changelog. For example, see [this changelog entry](https://github.com/tarantool/tarantool/blob/master/changelogs/unreleased/drop-net-box-console.md).

Reply:
done